src/js/components/PlacementConstraintsSchemaField.js

```diff
@@ -20,8 +20,35 @@ import CreateServiceModalFormUtil
 import MarathonAppValidators
   from "#PLUGINS/services/src/js/validators/MarathonAppValidators";
 
+import PlacementValidators
+  from "#PLUGINS/services/src/js/validators/PlacementsValidators";
+
 const jsonReducer = combineReducers({ constraints: JSONReducer });
 
+const InvalidJsonField = props => (
```

Comment:
Orlando said:
This looks like a good candidate for our component/pattern library, but we should rename the component to simply JsonField.

Reply:
Done.